v1.16-4.8/Dockerfile

```diff
@@ -30,6 +30,7 @@ RUN apk add --no-cache \
  && apk add --no-cache $BUILD_DEPS \
  && apk add --no-cache "libssl3>=3.1.4-r1" \
  && apk add --no-cache "libcrypto3>=3.1.4-r1" \
+ && apk add --no-cache "bash" \
```

Comment:
Why is bash required?

Reply:
> Why is bash required?

when I install rdkafka. It is required bash.

Reply:
bash has been moved to $BUILD_DEPS, after build it will be removed. @OverOrion